src/state/edit-content/actions.js

```diff
@@ -153,14 +152,14 @@ export const saveContent = values => (dispatch, getState) => new Promise((resolv
       firstEditor: currentUser,
       typeCode: getNewContentsType(state),
     };
-    dispatch(sendPostAddContent(requestObject)).then(() => resolve());
+    dispatch(sendPostAddContent(requestObject)).then(res => resolve(res));
   } else {
     const { id, typeCode } = getStateContent(state);
     const requestObject = {
       ...enhancedValues,
       id,
       typeCode,
     };
-    dispatch(sendPutEditContent(id, requestObject)).then(() => resolve());
+    dispatch(sendPutEditContent(id, requestObject)).then(res => resolve(res));
```

Comment:
`dispatch(sendPostAddContent(requestObject)).then(res => resolve(res));` can be easier with `dispatch(sendPostAddContent(requestObject)).then(resolve);`, right? you're still passing the `res` variable to whom it receives